app/src/main/res/drawable/add_profile_edit_text_background.xml

```diff
@@ -2,5 +2,5 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   <item android:drawable="@drawable/edit_text_black_border" android:state_focused="true" />
   <item android:drawable="@drawable/edit_text_black_border" android:state_pressed="true" />
-  <item android:drawable="@drawable/edit_text_background_border" />
+  <item android:drawable="@drawable/add_profile_edit_text_background_border" />
 </selector>
```

Comment:
This file is not getting used any where? Can you please confirm and delete this file.

Reply:
This file was earlier being used by previous input layout I guess. removing it